barretenberg/cpp/src/barretenberg/vm2/tracegen/sha256_trace.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <list>
```

Comment:
This import is not needed, I think it's a leftover from before using `::Container`.